java/core/src/main/java/co/worklytics/psoxy/storage/impl/ColumnarBulkDataSanitizerImpl.java

```diff
@@ -87,8 +87,9 @@ public void sanitize(@NonNull Reader reader,
             .setTrim(true)
             .build();
 
+        ParsedFirstLine parsedFirstLine = parseFirstLine(reader);
```

Comment:
I'd include a comment here about why use this and not `records.getFirstEndOfLine()`